mpd/errors.py

```diff
@@ -0,0 +1,15 @@
+# Imported from https://github.com/MusicPlayerDaemon/MPD/blob/master/src/protocol/Ack.hxx
```

Comment:
What kind of header should I add here?

Reply:
This is what we used:

```
# python-mpd2: Python MPD client library
#
# Copyright (C) 2021-  ....
#
# python-mpd2 is free software: you can redistribute it and/or modify
# it under the terms of the GNU Lesser General Public License as published by
# the Free Software Foundation, either version 3 of the License, or
# (at your option) any later version.
#
# python-mpd2 is distributed in the hope that it will be useful,
# but WITHOUT ANY WARRANTY; without even the implied warranty of
# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
# GNU Lesser General Public License for more details.
#
# You should have received a copy of the GNU Lesser General Public License
# along with python-mpd2.  If not, see <http://www.gnu.org/licenses/>.
```

Also switching to spdx-indentifier would be better.